src/components/Spawn/spawnHostModal/getFormSchema.tsx

```diff
@@ -520,11 +519,15 @@ export const getFormSchema = ({
               : "hidden",
           },
           volumeSelect: {
-            "ui:widget": isVirtualWorkstation ? AntdSelect : "hidden",
-            "ui:hideError": true,
+            "ui:widget": isVirtualWorkstation ? undefined : "hidden",
```

Comment:
[nit] Not part of your changes but some optional cleanup -- `shouldRenderVolumeSelection` already cases on `isVirtualWorkstation`, so I think we could just use that or `hidden` to conditionally render these.